pkg/custompluginmonitor/custom_plugin_monitor.go

```diff
@@ -135,7 +135,6 @@ func (c *customPluginMonitor) monitorLoop() {
 			}
 			glog.V(3).Infof("Receive new plugin result for %s: %+v", c.configPath, result)
 			status := c.generateStatus(result)
-			glog.Infof("New status generated: %+v", status)
```

Comment:
Can we keep this line with lower log level?

Reply:
Done